contrib/parseq-lambda-names/pom.xml

```diff
@@ -0,0 +1,192 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.linkedin.parseq</groupId>
+    <artifactId>parseq-lambda-names</artifactId>
+    <version>2.6.9-SNAPSHOT</version>
+
+    <dependencies>
+        <dependency>
+        <groupId>com.linkedin.parseq</groupId>
+            <artifactId>parseq</artifactId>
+            <version>2.6.9-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>com.ea.agentloader</groupId>
+            <artifactId>ea-agent-loader</artifactId>
+            <version>1.0.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm-all</artifactId>
+            <version>5.1</version>
+        </dependency>
+        <dependency>
```

Comment:
I am wondering if we should remove dependency on slf4j? The problem is that some slf4j binding is still required and it needs to be available on System Class Loader what might not always be the case. I am wondering if should replace all LOG.xxx with System.out?